src/Microsoft.OData.Core/WriterValidator.cs

```diff
@@ -6,9 +6,8 @@
 
 namespace Microsoft.OData
 {
-    using System;
     using System.Collections.Generic;
-#if NETSTANDARD2_0_OR_GREATER
+#if NETSTANDARD2_0_OR_GREATER || NETCOREAPP3_1_OR_GREATER

```

Comment:
Why doesn't `NETSTANDARD2_0_OR_GREATER` also include `NETCOREAPP_3_1+`?

Reply:
In the past #if NETSTANDARD2_0_OR_GREATER` used to apply to .NET Core 3.1 and later. But when we added `.netcoreapp3.1` as an explicit target framework in our csproj file, `NETSTANDARD2_0_OR_GREATER` stopped applying to it. I don't know why actually. But since then, we had to manually use `NETCOREAPP3_1_OR_GREATER`.